build.gradle

```diff
@@ -130,8 +130,10 @@ dependencies {
 
     // GCP
     compile 'com.jcraft:jsch:0.1.55'
-    compile 'com.google.cloud:google-cloud-os-login:2.0.13'
-    compile 'com.google.apis:google-api-services-compute:v1-rev20220322-1.32.1'
+    compile 'com.google.apis:google-api-services-compute:v1-rev235-1.25.0'
```

Comment:
v1-rev235-1.25.0 is very old version (released in year 2020) compared to already using version right. 
Latest is v1-rev20221026-2.0.0